src/main/java/com/linkedin/kmf/services/MultiClusterTopicManagementService.java

```diff
@@ -274,7 +276,7 @@ void maybeCreateTopic() throws Exception {
         NewTopic newTopic = new NewTopic(_topic, numPartitions, (short) _replicationFactor);
         newTopic.configs((Map) _topicProperties);
         CreateTopicsResult createTopicsResult = _adminClient.createTopics(Collections.singletonList(newTopic));
```

Comment:
Can you await the createTopics() result here? Then we wouldn't necessarily need retries on describing the topics because we would be sure the create topic future is completed.